web/gds-user-ui/src/index.css

```diff
@@ -6,6 +6,7 @@ body {
   height: '100%';
   font-family: 'Open sans', sans-serif;
   scroll-behavior: smooth;
+  overflow-anchor: none;
```

Comment:
While making updates to the form, I noticed that the page would often scroll to the center of the page instead of the top. This change appears to remove a browser default causing the issue. https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-anchor